dss/stepfunctions/__init__.py

```diff
@@ -5,6 +5,43 @@
 import boto3
 
 
+def step_functions_arn(state_machine_name_template: str) -> str:
+    """
+    The ARN of a state machine, with name derived from `state_machine_name_template`, with string formatting to
+    replace {stage} with the dss deployment stage.
+    :param state_machine_name_template:
+    :return:
```

Comment:
If you're adding signature docstrings, please fill them out. If your IDE has inserted a placeholder signature docstring but you don't want to fill it out, please delete it.